src/main/scala/fundamentals/level03/Exceptions2EitherExercises.scala

```diff
@@ -106,7 +106,8 @@ object Exceptions2EitherExercises {
     * scala> createValidPeople
     * = List(Person(Tokyo, 30), Person(Berlin, 43))
     *
-    * Hint: Use `map`, `createPerson` and `collect`
+    * Hint: Use `map` and `flatten`
+    * Hint: Use `toSeq` to convert Either into something that can be flattened with List.
```

Comment:
Can we use `toList`? Leave out `Seq` from this.

Reply:
scala weirdness, disallows this. 😞